lib/praxis/plugins/praxis_mapper_plugin.rb

```diff
@@ -103,7 +105,7 @@ def setup!
           unless log_stats == 'skip'
             Praxis::Notifications.subscribe 'praxis.request.all' do |name, *junk, payload|
               if (identity_map = payload[:request].identity_map)
-                PraxisMapperPlugin::Statistics.log(identity_map, log_stats)
+                PraxisMapperPlugin::Statistics.log(payload[:request], identity_map, log_stats)
```

Comment:
Any reason we need to pass in the request to this log method? Seems like we can short-circuit here, no?